index.d.ts

```diff
@@ -251,6 +254,66 @@ declare module 'gdax' {
         getTrailingVolume(): Promise<any>;
     }
 
+    export namespace WebsocketMessage {
+        export type Heartbeat = {
+            type: 'heartbeat'
+            sequence: number
+            last_trade_id: number
+            product_id: string
+            time: string // ISO Date string without time zone
+        }
+        export type L2Snapshot = {
+            type: 'snapshot'
+            product_id: string
+            bids: [string, string][] // strings are serialized fixed-point numbers
+            asks: [string, string][]
+        }
+        export type L2Update = {
+            type: 'l2update'
+            product_id: string
+            changes: [string, string, string][] // [side, price, new size]
```

Comment:
Since you're here, you could add a Side type and change all 'side: string' to 'side: Side'.

export declare type Side = 'buy' | 'sell';